lib/uplink/uplink.go

```
@@ -68,6 +78,11 @@ func (c *Config) setDefaults(ctx context.Context) error {
 	if c.Volatile.MaxInlineSize.Int() == 0 {
 		c.Volatile.MaxInlineSize = 4 * memory.KiB
 	}
+	if c.Volatile.MaxMem.Int() == 0 {
+		c.Volatile.MaxMem = 4 * memory.MiB
+	} else if c.Volatile.MaxMem.Int() < 0 {
+		c.Volatile.MaxMem = 0
```

Or should we just return an error here?